LFPy/eegmegcalc.py

```diff
@@ -0,0 +1,476 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan 15 17:24:00 2015
+
+@author: solveig
+"""
+
+from __future__ import division
+from scipy.special import eval_legendre, lpmv
+import numpy as np
+
+class EEGMEGCalc:
```

Comment:
I think a more appropriate name for this class would be ```FourSphereVolumeConductor``` (then we later on add ```OneSphereVolumeConductor``` and ```InfiniteVolumeConductor```). I'm not sure if these classes may benefit from some class inheritance structure yet, so let's keep these separated. 

Class definitions should be inherited from the base ```object``` class, like ```class FourSphereVolumeConductor(object):```

Can you also add numpydoc style "Parameters" (types, units, description) and "Example" blocks below such that  class usage becomes clear?